src/components/DeleteBooks/DeleteBooks.js

```diff
@@ -0,0 +1,194 @@
+        id: doc.id,
+        title: doc.data().title,
+        author: doc.data().author,
+        description: doc.data().description,
+        thumbnail: doc.data().thumbnail,
+      }));
+
+      const resultsAuthor = snapshotAuthor.docs.map((doc) => ({
+        collection: collectionName,
+        id: doc.id,
+        title: doc.data().title,
+        author: doc.data().author,
+        description: doc.data().description,
+        thumbnail: doc.data().thumbnail,
+      }));
+
+      const resultsId = snapshotId.docs.map((doc) => ({
+        collection: collectionName,
+        id: doc.id,
+        title: doc.data().title,
+        author: doc.data().author,
+        description: doc.data().description,
+        thumbnail: doc.data().thumbnail,
+      }));
+
+      results.push(...resultsTitle, ...resultsAuthor, ...resultsId);
+    }
+
+    setSearchResults(results);
+  };
+
+  useEffect(() => {
+    fetchBooks();
+  }, [searchQuery, selectedCollections]);
+
+  const deleteBook = async (bookId) => {
+    for (const collectionName of selectedCollections) {
+      await deleteDoc(doc(db, collectionName, bookId));
+    }
+    fetchBooks();
+  };
+
+  const handleSearch = () => {
+    fetchBooks();
+  };
+
+  const handleCheck = (event, bookId) => {
+    if (event.target.checked) {
+      setChecked((prevChecked) => [...prevChecked, bookId]);
+    } else {
+      setChecked((prevChecked) => prevChecked.filter((id) => id !== bookId));
+    }
+  };
+
+  const handleDelete = async () => {
+    for (const collectionName of selectedCollections) {
+      const checkedBooks = searchResults.filter((result) =>
+        checked.includes(result.id) && result.collection === collectionName
+      );
+
+      for (const book of checkedBooks) {
+        await deleteDoc(doc(db, collectionName, book.id));
+      }
+    }
+
+    setChecked([]);
+    fetchBooks();
+  };
+
+  return (
+    <div>
+      <Navigation
+        name={name}
+        setName={setName}
+        verified={verified}
+        setVerified={setVerified}
+        employee={employee}
+        setEmployee={setEmployee}
+        signOn={signOn}
+        setSignOn={setSignOn}
+        displayCart={displayCart}
+        setDisplayCart={setDisplayCart}
+        users={users}
+        setUsers={setUsers}
+        search={search}
+        setSearch={setSearch}
+        books={books}
+        setBooks={setBooks}
+        bookCategory={bookCategory}
+        setBookCategory={setBookCategory}
+      />
+      <LogonForm
+        signOn={signOn}
+        setSignOn={setSignOn}
+        setVerified={setVerified}
+        employee={employee}
+        setEmployee={setEmployee}
+        users={users}
+        setUsers={setUsers}
+      />
+      <h1>Delete Books</h1>
+
+      <ul>
+        {selectedCollections.map((collectionName) => (
+          <li key={collectionName}>
+            <button onClick={() => fetchBooks(collectionName)}>
+              {collectionName}
+            </button>
+          </li>
```

Comment:
why dont we set these to radio buttons instead of standard buttons that way you can see what is selected and which category your searching